doc/sphinx-guides/source/developers/dependencies.rst

```diff
@@ -221,10 +225,10 @@ on *Z*. You definitely don't want to head down that road.
 
 **Follow the rules to be safe:**
 
-1. Do **not use transitive deps implicit**: add a direct dependency for transitive deps you re-use in your code.
```

Comment:
This is for line 202 above but I find "the dependent version of the nearest version wins" unclear.

Reply:
I edited the file and added a comment above. Let's discuss further changes there. Feel free to add suggestions :smile: